android/libraries/rib-base/src/main/java/com/badoo/ribs/android/IntentCreator.kt

```diff
@@ -0,0 +1,10 @@
+package com.badoo.ribs.android
+
+import android.content.Intent
+
+interface IntentCreator {
+
+    fun create(): Intent
```

Comment:
Will it be better to have one method with default parameter null?
It's quite common practice to use default parameters instead of overloading.

Reply:
From the perspective of client code calling it, it will look the same. From the perspective of class implementing this interface, it's either two methods or if branching inside one. I don't see the big benefit?

Reply:
I feel that it is better to keep the amount of methods to minimal
The benefit is negligible, but the API is clearer

Reply:
You right, it will only affect `RibActivity` which is not users one.
I thought it was on the user's side.
I agree with the current approach.

Reply:
Fixed.